### PR TITLE
department messenger bags

### PR DESCRIPTION
**Pull request by SonixApache**
🆑 
 - tweak: Added messenger bags to the departmental backpack spawn lists
 - tweak: Added satchels and messenger bags to departmental wardrobes where appropriate

### CODE REVIEW

File: __DEFINES/__compile_options.dm [hunk 1]
```diff
@@ -25,14 +25,14 @@
 // test_box.dm:
 //#define MAP_OVERRIDE 5
 // test_tiny.dm:
-//#define MAP_OVERRIDE 6
+#define MAP_OVERRIDE 6
```
Comment (MadmanMartian) [inline __DEFINES/__compile_options.dm]: !

File: __DEFINES/__compile_options.dm [hunk 2]
```diff
@@ -25,14 +25,14 @@
...
 // tgstation.dm:
 //#define MAP_OVERRIDE 7
 
 
 // Toggles several features, explained in their respective comments.
 // You can turn those on and off manually if you prefer, instead of setting this
-#define DEVELOPER_MODE 0
+#define DEVELOPER_MODE 1
```
Comment (MadmanMartian) [inline __DEFINES/__compile_options.dm]: !!

File: vgstation13.dme [hunk 3]
```diff
@@ -2449,7 +2449,7 @@
 #include "maprendering\maprendering.dm"
 #include "maps\_map.dm"
 #include "maps\_map_override.dm"
-#include "maps\tgstation.dm"
+#include "maps\packedstation.dm"
```
Comment (MadmanMartian) [inline vgstation13.dme]: !!!

File: __DEFINES/__compile_options.dm (hunk 1 above)
Comment (SonixApache) [reply __DEFINES/__compile_options.dm]: goddamnit

**Review by MadmanMartian — APPROVED** [review]

**Pull request merged**